torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4484,6 +4500,16 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                # AssertionError: UserWarning not triggered : Resized a non-empty tensor but did not warn about it.
                SkipInfo('TestCommon', 'test_out', dtypes=[torch.float32]),
            )),
+    OpInfo('logcumsumexp',
+           dtypes=floating_types_and(),
+           dtypesIfCUDA=floating_types_and(torch.half),
+           skips=(
+               # AssertionError: UserWarning not triggered : Resized a non-empty tensor but did not warn about it.
+               SkipInfo('TestCommon', 'test_out', dtypes=(torch.float32,), device_type='cuda'),
+               # logcumsumexp_backward not implemented for 'Half
+               SkipInfo('TestOpInfo', 'test_supported_backward', dtypes=(torch.bfloat16, torch.float16), device_type='cuda'),
```

Comment:
Nit: the bfloat16 is superfluous here because it doesn't appear in the CUDA dtypes list